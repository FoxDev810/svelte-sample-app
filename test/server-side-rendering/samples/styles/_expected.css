div.svelte-724714405,.svelte-724714405 div{color:red}