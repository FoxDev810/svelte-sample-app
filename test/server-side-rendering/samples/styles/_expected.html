<div svelte-724714405>red</div>